config/checkstyle/micrometer-style.xml

```diff
@@ -0,0 +1,336 @@
+<profiles version="13">
```

Comment:
I'm a little nervous about this coming out of Eclipse. But I suppose we can test the validity of formatting by applying to the whole project, then opening IntelliJ and formatting the whole project. Should be zero changes on that last step.

Reply:
Yes, that was a plan. And as a I said, there are still minor differences that I'm working on.